features/git-town/configuration/configuration_reset.feature

```diff
@@ -0,0 +1,36 @@
+Feature: resetting the configuration
+
+  As a user wishing to remove all Git Town settings from a repository
+  I want to be able to cleanly reset the Git Town configuration with a single command
+  So that I can continue working on actual work and stay productive
+
+
+  Scenario: everything is configured
+    Given I have configured the main branch name as "main"
+    And my non-feature branches are configured as "qa" and "staging"
+    When I run `git town config --reset`
+    Then I see "Your Git Town settings have been reset for this repository"
```

Comment:
Along the lines of successful commands produce no output, I would vote that this produces no output.